### PR TITLE
invite roles comes from registered-roles API

### PR DESCRIPTION
**Pull request by HarshOB**

### CODE REVIEW

File: client/src/pages/flow/List.vue [hunk 1]
```diff
@@ -763,22 +763,50 @@ export default {
...
-          roles.push({
-            value1: temp1[index],
-            label1: temp1[index]
-          })
-          this.options = roles
+    async showInviteDialog (query) {
+      // if (query.row.roles !== undefined) {
+      //   this.flowId = 'workflow_' + query.row.id
+      //   let temp1 = query.row.roles.split(',')
+      //   let roles = []
```
Comment (chiragob) [inline client/src/pages/flow/List.vue]: remove commented code and console log

File: client/src/pages/flow/List.vue [hunk 2]
```diff
@@ -763,22 +763,50 @@ export default {
...
+        console.log(response)
+        if (response.data.data.length > 0) {
+          let roles = []
+          for (let i = 0; i < response.data.data.length; i++) {
+            roles.push({
+              value1: response.data.data[i].role,
+              label1: response.data.data[i].role
+            })
+          }
+          console.log('roles', roles)
```
Comment (chiragob) [inline client/src/pages/flow/List.vue]: console log

Comment (HarshOB) [reply client/src/pages/flow/List.vue]: done

File: client/src/pages/flow/List.vue (hunk 1 above)
Comment (HarshOB) [reply client/src/pages/flow/List.vue]: done

**Pull request merged**